scripts/check-tools.sh

```
@@ -0,0 +1,63 @@
+set -e
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+NC='\033[0m'
+BOLD='\033[1m'
+
+ERRORS=0
+
+check() {
+  local name="$1"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)